format/view-spec.md

```diff
@@ -111,12 +111,13 @@ The SQL representation stores the view definition as a SQL SELECT, with metadata
 |-------------|---------------------|----------------|-------------|
 | _required_  | `type`              | `string`       | Must be `sql` |
 | _required_  | `sql`               | `string`       | A SQL SELECT statement |
-| _required_  | `dialect`           | `string`       | The dialect of the `sql` SELECT statement (e.g., "trino" or "spark") |
+| _required_  | `dialect`           | `string`       | The dialect of the `sql` SELECT statement (e.g., "trino" or "spark"). Unless a version is provided only one view definition for each `type` and `dialect` should be present in a view definition.  |
+| _optional_  | `dialect_version`   | `string`       | A version to provide additional information for consumers of the view. It is permissible to have multiple versions of the same dialect in a view definition.  |
 | _optional_  | `schema-id`         | `int`          | ID of the schema produced by the SELECT statement |
 | _optional_  | `default-catalog`   | `string`       | Catalog name to use when a reference in the SELECT does not contain a catalog |
 | _optional_  | `default-namespace` | `list<string>` | Namespace to use when a reference in the SELECT is a single identifier |
-| _optional_  | `field-aliases`     | `list<string>` | Column names optionally specified in the create statement |
-| _optional_  | `field-comments`    | `list<string>` | Column descriptions (COMMENT) optionally specified in the create statement |
+| _optional_  | `field-aliases`     | `list<string>` | Column names optionally specified in the create statement. When present SQL queries should be resolved against these instead of the column names in the schema. |
+| _optional_  | `field-comments`    | `list<string>` | Column descriptions (COMMENT) optionally specified in the create statement. When present systems should prefer displaying these as documentation instead of the documentation in the schema. |
```

Comment:
I don't think these statements are correct. The schema should already have these applied so that the engine doesn't need to go to a particular representation and update it.

The purpose of these fields is to capture the column names and comments from the original `CREATE VIEW` statement.

Reply:
What isn't clear to me is how we expect multiple dialects to be published here?  I updated with my how interpreted your comments, I apologize if this is still incorrect.